LongWarOfTheChosen/Src/WotCWeaponsAndArmor/Classes/X2Item_Armors.uc

```diff
@@ -0,0 +1,265 @@
+// This is an Unreal Script
+class X2Item_Armors extends X2Item;

```

Comment:
How about `X2Item_FactionArmors`?

Reply:
Ok I will rename to X2Item_FactionArmors.

Reply:
This has been renamed.